jolly-jellyfish/src/page_maker/templates/page_maker/webpage_detail.html

```diff
@@ -0,0 +1,30 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<div><h1>{{ webpage.user_title }}</h1></div>
```

Comment:
```suggestion
<h1><span><span>{{ webpage.user_title }}</span></span></h1>
```
This will have the custom theme, right?